docs/DeveloperGuide.adoc

```diff
@@ -1044,10 +1052,12 @@ Use case resumes at step 1.
 
 *MSS*
 
-1. User requests to list students.
-2. TA-Tracker shows a list of students.
-3. User requests to edit a specific student in the list.
-4. TA-Tracker edits the student according to the specified parameters.
+. User requests to list students.
+. TA-Tracker shows a list of students at the student tab.
+. User filters the list in order to get the student's index.
```

Comment:
Maybe can assume user is already in student tab (like include a see use case #)
Cos hard to explain filtering in the current use case.

Reply:
Matric num not index.